recipes/wtl/all/conanfile.py

```diff
@@ -0,0 +1,24 @@
+from conans import ConanFile, tools
+
+import os
+
+class WTLConan(ConanFile):
+    name = "wtl"
+    license = "Microsoft Public License"
```

Comment:
```suggestion
    license = "MS-PL"
```

Reply:
Fixed in 6c6f4775993f0e672ac9e55484093c6237c2d215